pywatts/core/pipeline.py

```diff
@@ -120,9 +120,9 @@ def _collect_batches(self, last_steps):
             self.counter += self.batch
         return result
 
-    def _collect_results(self, inputs):
+    def _collect_results(self, inputs, batch=False):
```

Comment:
I think there is a risk of confusing self.batch and batch. Consider renaming the input `batch` to `use_batch` or something. Then, line 100, 108, 110, 286, and 324 need to be checked/adapted.